geoh5py/groups/property_group.py

```diff
@@ -236,18 +236,22 @@ def remove_properties(self, data: Data | list[Data | uuid.UUID] | uuid.UUID):
         if self._properties is None:
             return
 
+        update = False
         for elem in data:
-            if isinstance(elem, uuid.UUID):
-                entity = self.parent.get_entity(elem)[0]
-            elif isinstance(elem, Data) and elem in self.parent.children:
-                entity = elem
+
+            if isinstance(elem, Data):
```

Comment:
isn't a bit repetitive with line 247
get uid then get the data ?

Reply:
I can simplify.